src/cr/cube/cubepart.py

```diff
@@ -1435,6 +1458,19 @@ def smoothed_dimension_dict(self):
         """dict, row dimension definition"""
         return self._rows_dimension._dimension_dict
 
+    @lazyproperty
+    def share_sum(self):
```

Comment:
I'm starting to think if we should have a base class for both strand and slice... Maybe also start adding API into the name of those classes, because that's essentially what they are. Maybe not now, but just a thought.